services/business/istio_validations.go

```diff
@@ -108,6 +108,8 @@ func (in *IstioValidationsService) GetIstioObjectValidations(namespace string, o
 			istioDetails.DestinationRules = []kubernetes.IstioObject{dr}
 			objectCheckers = []ObjectChecker{noServiceChecker}
 		}
+	case "serviceentries":
+		// Validations on ServiceEntries are not yet in place
```

Comment:
Should we enter a JIRA right now for it, or is it something that may even not be necessary to implement?

Reply:
I don't think we will extend too much validations for TP perspective.
I guess important ones are in place as validations per serviceentries and gateways need more "business" context to automate if something is good or not.
I wouldn't add them yet, I think.
Also, in the validations side we are putting an eye into Kiali.
Today it started "some" validation but more technical, without context (just to check if there is some missing field).
I think that we would like to integrate what kiali does in the future, and just to use Kiali as presentation + adding additional business context validation that might have sense (weight, missing services, etc).